### PR TITLE
ebu128 version

### PR DESCRIPTION
**Pull request by egornovivan**

### CODE REVIEW

**Comment by burner1024** [comment]
What's the effective difference, in layman's terms? Need to explain in readme, so that users can make the choice.

**Comment by egornovivan** [comment]
> 
> 
> What's the effective difference, in layman's terms? Need to explain in readme, so that users can make the choice.

uhq - lossless 44khz, files loudnes variable(-14 to -20 LU etc)
hq - lossy 22khz, files loudnes ≈ -16 LU
uhq_ebur128 - lossy 44khz, files loudnes ≈ -23 LU approx as in original game files.
hq_ebur128 - lossy 22khz, files loudnes ≈ -23 LU approx as in original game files.

Some sound cards may have problems with hq/uhq versions

**Comment by burner1024** [comment]
That's not exactly layman's. What I mean is a direction: "If you want this - use this version, otherwise - use that one".

**Comment by egornovivan** [comment]
> "If you want this - use this version, otherwise - use that one".

If you need maximum quality, use hq or uhq versions
If you hear crackling in the speakers, then use the hq_ebur128 or uhq_ebur128 versions

**Comment by burner1024** [comment]
Got it. Thank you.
